views/js/qtiCreator/editor/styleEditor/styleSheetToggler.js

```diff
@@ -33,16 +33,35 @@ define([
 
         var init = function (itemConfig) {
 
+            const _createInfoBox = function (data) {
+                var $messageBox = $(genericFeedbackPopup(data)),
+                    closeTrigger = $messageBox.find('.close-trigger');
+
+                $('body').append($messageBox);
+
+                closeTrigger.on('click', function () {
+                    $messageBox.fadeOut(function () {
+                        $(this).remove();
+                    });
+                });
+
+                setTimeout(function () {
```

Comment:
Why we need SetTimeout in 4523 milliseconds?

Reply:
This setTimeout was already there, and I just moved the function before it was calling. I had a linter issue with that.
I understand that it is forcing the modal to close if the user hasn't closed it after a while.

Reply:
can you change it to more 'beautiful ' number? like 4000?